SMLHelper/Options/KeybindModOption.cs

```diff
@@ -139,9 +140,16 @@ internal override void AddToPanel(uGUI_TabbedControlsPanel panel, int tabIndex)
             // Update bindings
             binding.device = Device;
             binding.value = KeyCodeUtils.KeyCodeToString(Key);
+#if SUBNAUTICA
             binding.onValueChanged.RemoveAllListeners();
             var callback = new UnityAction<KeyCode>((KeyCode key) => parentOptions.OnKeybindChange(Id, key));
             binding.onValueChanged.AddListener(new UnityAction<string>((string s) => callback?.Invoke(KeyCodeUtils.StringToKeyCode(s))));
+#elif BELOWZERO
+            binding.action = ButtonPatcher.EnsureButton(Label, KeyCodeUtils.KeyCodeToString(Key), Device);
```

Comment:
I stumbled upon those changes in `uGUI_Binding` when I tried to port my hotkeys mod to BZ. Did you try to do this without patching enums? I'm still testing my version, but it seems to work with less code.